ci/release/update-version.sh

```diff
@@ -47,3 +47,8 @@ sed_runner 's/PROJECT_NUMBER         = .*/PROJECT_NUMBER         = '${NEXT_FULL_
 # sphinx docs update
 sed_runner 's/version = .*/version = '"'${NEXT_SHORT_TAG}'"'/g' docs/source/conf.py
 sed_runner 's/release = .*/release = '"'${NEXT_FULL_TAG}'"'/g' docs/source/conf.py
+
+# bump cudf
+for FILE in conda/environments/*.yaml dependencies.yaml; do
+  sed_runner "s/cudf=.*/cudf=${NEXT_SHORT_TAG}/g" ${FILE};
+done
```

Comment:
This script will also need to bump the `FORMAT_FILE_URL`'s use of `branch-23.02` in `check_style.sh` after applying my other suggestion. See cudf for reference.

Reply:
added, thanks!